doc/conda/Jenkinsfile.conda.template

```diff
@@ -4,13 +4,13 @@ pipeline {
             image 'lsstts/conda_package_builder:latest'
             alwaysPull 'true'
             args "--env {csc_config_repo}_DIR=/home/saluser/{csc_config_repo} --env LSST_DDS_DOMAIN=citest -u root --entrypoint=''"
-            # csc_config_repo should be changed to the CSC's configuration repo name i.e. ts_config_attcs for the athexapod.
-            # These could be changed to environment variables but that has not been tested yet, so does not make it into the form of the template.
+            // csc_config_repo should be changed to the CSC's configuration repo name i.e. ts_config_attcs for the athexapod.
+            // These could be changed to environment variables but that has not been tested yet, so does not make it into the form of the template.
         }
     }
     environment {
         package_name = "{csc_name}"
-        # name of the csc package i.e. "ts_athexapod"
+        // name of the csc package i.e. "ts_athexapod"
```

Comment:
so, the `ts` part of the name is added later on. Can you update the comment to something on the lines of:

> Name of the csc package as specified in the meta.yaml file without the trailing `ts-`.

Reply:
Either that or remove the `ts-` in line 84, not sure what we agreed upon in the hackathon